diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/EmptyBlock.kt

```diff
@@ -41,12 +37,26 @@ class EmptyBlock(configRules: List<RulesConfig>) : DiktatRule(
         checkEmptyBlock(newNode, configuration)
     }
 
-    @Suppress("UnsafeCallOnNullableType")
+    private fun isEmptyWhiteSpace(node: ASTNode) =
```

Comment:
The method's name is confusing. Why does it return `true` if there is no white space? Also, it doesn't check for empty white space, it checks for newlines.